qa/tasks/cephfs/test_openfiletable.py

```diff
@@ -31,7 +32,7 @@ def test_max_items_per_obj(self):
 
         # Hold the file open
         file_count = 8
-        for i in xrange(0, file_count):
+        for i in range(0, file_count):
```

Comment:
file_count is 8, there is no need to use `xrange` and `six.moves.range` correspondingly

Reply:
ack. fixed.